server/lib/queries.js

```diff
@@ -1161,16 +1161,16 @@ const getTagFrequencies = async args => {
   let searchTermFragment = '';
   let term = args.searchTerm;
   // Cleanup term
+  term = sanitizeSearchTermForILike(trimSearchTerm(term));
   if (term && term.length > 0) {
-    term = sanitizeSearchTermForILike(trimSearchTerm(term));
     if (term[0] === '@') {
```

Comment:
Same stuff here, I don't get why we're moving that out of the `if (term`

Reply:
This is a mistake on my part I think, we should have this within. 🤔 I've corrected that. 👍🏽